anitya/config.py

```diff
@@ -21,6 +21,7 @@
 
 from datetime import timedelta
 import logging
+import logging.config
```

Comment:
This change seems unrelated to the commit. Is it needed? I recommend making a separate commit with it if it is.